From 289eae69721fadb4297d63b5eead6f5ea5e29c7b Mon Sep 17 00:00:00 2001
From: Jenny TC <jenny.tc@intel.com>
Date: Mon, 20 Oct 2014 17:25:34 +0530
Subject: [PATCH 1/1] PMIC-BCU: fix config option for Wcove BCU

Whiskey Cove BCU should get compiled if INTEL_SOC_PMIC option is
selected.

Change-Id: Ib9ba8c78fe9a55e00e56d53c337bbd5cc10bd7cc
Signed-off-by: Jenny TC <jenny.tc@intel.com>
---
 drivers/hwmon/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hwmon/Makefile b/drivers/hwmon/Makefile
index 480217cf2b24..b3a369348ad7 100644
--- a/drivers/hwmon/Makefile
+++ b/drivers/hwmon/Makefile
@@ -142,7 +142,7 @@ obj-$(CONFIG_SENSORS_W83L785TS)	+= w83l785ts.o
 obj-$(CONFIG_SENSORS_W83L786NG)	+= w83l786ng.o
 obj-$(CONFIG_SENSORS_WM831X)	+= wm831x-hwmon.o
 obj-$(CONFIG_SENSORS_WM8350)	+= wm8350-hwmon.o
-obj-$(CONFIG_INTEL_PMIC_WCOVE)	+= intel_wcove_bcu.o
+obj-$(CONFIG_INTEL_SOC_PMIC)	+= intel_wcove_bcu.o
 
 obj-$(CONFIG_PMBUS)		+= pmbus/
 
-- 
1.7.9.5

